src/PhoneValidation.scss

```diff
@@ -0,0 +1,46 @@
+#root {
```

Comment:
I'd recommend using class names instead of id tags to style. Following the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single-responsibility_principle), I'd recommend using ids and classnames both for styling.

My recommendation would be to use classnames for styling and ids for scripting